packages/account/src/Sections/Verification/ProofOfIncome/proof-of-income-container.jsx

```diff
@@ -0,0 +1,84 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { income_status_codes, poinc_documents_list } from './proof-of-income-utils';
+import { Loading, useStateCallback } from '@deriv/components';
+import { WS } from '@deriv/shared';
+import PoincUnverified from 'Components/poinc/status/unverified';
+import PoincReceived from 'Components/poinc/status/received';
+import PoincLimited from 'Components/poinc/status/limited';
+import PoincVerified from 'Components/poinc/status/verified';
+import PoincNotRequired from 'Components/poinc/status/not-required';
+import ProofOfIncomeForm from './proof-of-income-form.jsx';
+import { populateVerificationStatus } from '../Helpers/verification';
+
+const ProofOfIncomeContainer = ({ is_switching, refreshNotifications }) => {
+    const [is_loading, setIsLoading] = React.useState(true);
+    const [authentication_status, setAuthenticationStatus] = useStateCallback({
+        allow_document_upload: false,
+        needs_poinc: false,
+        income_status: null,
+        is_age_verified: false,
+    });
+
+    React.useEffect(() => {
+        if (!is_switching) {
+            WS.authorized.getAccountStatus().then(response => {
+                const { get_account_status } = response;
+                const { allow_document_upload, income_status, needs_poinc, is_age_verified } =
+                    populateVerificationStatus(get_account_status);
+
+                setAuthenticationStatus(
+                    {
+                        ...authentication_status,
+                        ...{
+                            allow_document_upload,
+                            needs_poinc,
+                            income_status,
+                            is_age_verified,
+                        },
+                    },
+                    () => {
+                        setIsLoading(false);
+                        refreshNotifications();
+                    }
+                );
+            });
+        }
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [is_switching]);
+
+    const onSubmit = () => {
```

Comment:
can we merge these to :
```suggestion
    const onSubmit = (status) => {
          setAuthenticationStatus({ ...authentication_status, ...{ income_status: status } });
    };
```
what do you think?

Reply:
will change it, and add prop in func call in form component